django_rq/management/commands/rqclean.py

```diff
@@ -0,0 +1,25 @@
+from django.core.management.base import BaseCommand
+
+from django_rq import get_queue
+
+
+class Command(BaseCommand):
```

Comment:
Given the nature of this command, I think we should prompt the user whether really wants to remove all jobs from the named queue. 

Would `rqempty` or `rqflush` be more descriptive than `rqclean`?